client/src/components/App.jsx

```diff
@@ -113,14 +117,23 @@ class App extends React.Component {
     });
   }
 
+  showAllItems() {
+    const updatedTabs = this.state.tabs.slice();
+    updatedTabs[this.state.view].showAll = true;
+    this.setState({
+      tabs: updatedTabs
+    });
+  }
+
   render() {
     return (
       <div>
         <GlobalStyle />
         <h1>Get the full experience and book a tour</h1>
         <Container>
           <TabBar tabs={this.state.tabs} currentTab={this.state.view} updateView={this.updateView}/>
-          {this.state.dataLoaded ? <Display tab={this.state.tabs[this.state.view]}/> : null}
+          {this.state.dataLoaded ? <Display tab={this.state.tabs[this.state.view]} showAllItems={this.showAllItems}/>
+          : null}
```

Comment:
Is it rendering slowly or are you just worried it looks messy? I think ternary operators are a good call

Reply:
more just worried it looks messy. thanks!